plaid/accounts.go

```diff
@@ -0,0 +1,102 @@
+package plaid
+
+import (
+	"encoding/json"
+	"errors"
+)
+
+type Account struct {
+	ID      string `json:"account_id"`
```

Comment:
nit: rename to `AccountID` to be consistent with the keyname returned from the API

Reply:
fixed!